xml/pop_banners.xml

```diff
@@ -238,7 +238,15 @@ matched against these patterns to fingerprint POP3 servers.
       <param pos="0" name="os.device" value="General"/>
       <param pos="1" name="os.version"/>
       <param pos="2" name="host.name"/>
-   </fingerprint>
+    </fingerprint>
+
+    <fingerprint pattern="^Hello there\.$">
```

Comment:
I checked the source and all instances of "Hello there." when googling around for stuff about mail/IMAP/POP led me to believe that this was Courier.